pkg/sql/logictest/testdata/logic_test/fk

```diff
@@ -4166,3 +4174,42 @@ create table t104546_fk_src (a int primary key, b int);
 
 statement error pgcode 42703 column "b" does not exist
 alter table t104546 add constraint con foreign key (b) references t104546_fk_src(b);
+
+statement ok
+create schema sc1;
+create schema sc2;
+create table sc1.parent (p int primary key);
+create table sc2.child(p int primary key, r int REFERENCES sc1.parent (p));
+
+query TTTTTT rowsort
+SELECT constraint_catalog, constraint_schema, constraint_name, unique_constraint_catalog, unique_constraint_schema, unique_constraint_name
+FROM  information_schema.referential_constraints;
```

Comment:
nit: could you add a WHERE clause to this so it only considers `unique_constraint_schema = sc1`? the other rows in the output adds a bit of noise that makes it harder to see what this test really cares about